commons/src/main/java/fr/free/nrw/commons/upload/UploadController.java

```diff
@@ -140,7 +140,8 @@ protected Contribution doInBackground(Void... voids /* stare into you */) {
             @Override
             protected void onPostExecute(Contribution contribution) {
                 super.onPostExecute(contribution);
-                uploadService.queue(UploadService.ACTION_UPLOAD_FILE, contribution);
+                //temporarily disabled for debugging
```

Comment:
Oh, okay! How do you do that, do you just leave that change off of every commit you make?